docs/translations.md

```diff
@@ -179,7 +179,7 @@ In case you have added a new language translation file and are pulling translati
 in sync. Running the following command in the project root
 
 ```
-node translations.js
+node scripts/translations.js
```

Comment:
Actually, it would make sense to create package.json script out of these `yarn run translations` or maybe even `yarn run translate`.